.eslintrc

```diff
@@ -90,10 +90,10 @@
     "no-use-before-define": 2,
     "radix": 2,
     "curly": 2,
-    "no-fallthrough": 0,
-    "default-case": 0,
-    "no-plusplus": 0,
-    "dot-notation": 0
+    "no-fallthrough": 2,
+    "default-case": 2,
+    "no-plusplus": 2,
```

Comment:
++ is very useful in things like for loops to keep things compact, as well as just a simple increment. However yes I agree we need to kill all those who pass the increment as a parameter!